glossary.rst

```diff
@@ -84,6 +117,41 @@ throughout the various guides and docs on our sites.
   Maximum values that are enforced across jobs, queues, and users, such as time
   and/or resource limits.
 
+.. _queue:
+.. _job_queue:
+
+**Queue**
+  A container for a set of jobs in priority order.  Queues may differ in what
```

Comment:
I think some might read this as "linux container" but I don't think that's what you mean. Is there another term we can use?

Reply:
Hmmm.  I used "container" b/c that's what we use in the RFC.   Doing nothing more than googling "what is a job queue", first description seems to use "data structure", which I don't think is quite right for this context.

Will ponder, perhaps could re-word the whole thing in a different way.

Reply:
Maybe something simple and stupid like a list? Or another synonym for queue?

Reply:
none of the synonyms spoke to me (vessel, receptacle, ...).  As I went for a walk, "data structure" began sounding better to me.

"A scheduling data structure that contains sets of jobs in priority order."

Reply:
Went for a walk, you say? I have a song for you! https://youtu.be/dZX6Q-Bj_xg

I think data structure sounds ok too, at least I won't confuse it with "those other container things."